scripts/prepare-ci-environment-k3d.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -e
+
+# UNAME should be darwin or linux
+UNAME="$(uname | tr "[:upper:]" "[:lower:]")"
+
+# EPINIO_BINARY is used to execute the installation commands
+EPINIO_BINARY="./dist/epinio-"${UNAME}"-amd64"
```

Comment:
Maybe use:

```
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```
and use that to refer to the binary? This way it doesn't matter where you invoke this script from.

Reply:
Apparently we already do that in other scripts. I would be content with just calling the scripts from the top level dir only, it's not user facing.

Reply:
fair